packages/Babylonian-Core.package/BPExampleToTest.class/instance/canBeCreated.in..st

```diff
@@ -0,0 +1,9 @@
+helper
+canBeCreated: aMethodName in: aClass
```

Comment:
This is interesting: This works as the combination of class, selector, and example name is unique. This is an assumption we might want to document here.